src/modules/auth/header.js

```diff
@@ -35,9 +35,13 @@ function HeaderComponent(props) {
 	let user_auth_url = App.getServiceURL('seacat_auth_webui');
 	// Get access control URL
 	let access_control_url = window.location.protocol + '//' + window.location.host + '#/auth/access-control';
-	let active_tenant = props.current;
-	if (active_tenant) {
-		access_control_url = window.location.protocol + '//' + window.location.host + '/?tenant=' + active_tenant + '#/auth/access-control';
+	// Check if Tenant service is available to get the access control URL with tenant
+	if (App.Services.TenantService) {
+		const params = new URLSearchParams(window.location.search);
+		let currentTenant = params.get('tenant');
```

Comment:
`App.Services.TenantService.get_current_tenant()`

Reply:
Refactored accordingly in those commits: https://github.com/TeskaLabs/asab-webui/pull/105/commits/a3e091bca806fe6678877a21704ee2c89b462f26 and https://github.com/TeskaLabs/asab-webui/pull/105/commits/fdf92de20247d7a2a9d723ed3df0af4a17d8746f